src/dos/dos_programs.cpp

```diff
@@ -95,6 +95,23 @@ void DOS_SetupPrograms(void)
 	MSG_Add("MSCDEX_UNKNOWN_ERROR","MSCDEX: Failure: Unknown error.\n");
 	MSG_Add("MSCDEX_WARNING_NO_OPTION", "MSCDEX: Warning: Ignoring unsupported option '%s'.\n");
 
+	MSG_Add("SHELL_CMD_RESCAN_HELP_LONG",
+	        "Clears the caches of a mounted drive.\n"
+	        "\n"
+	        "Usage:\n"
+	        "  \033[32;1mrescan\033[0m \033[36;1mDRIVE\033[0m\n"
+	        "  \033[32;1mrescan\033[0m [/a]\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[36;1mDRIVE\033[0m is the drive to clear caches.\n"
```

Comment:
Maybe `scan for changes` instead of `clear caches`?

Reply:
Thanks for the suggestion. DOSBox (Staging) originally used the word "clear/update cache", so I followed. But I agree the word may be a bit technical. So I fixed all references to these terms to "scan for changes".